nautilus_trader/live/config.py

```diff
@@ -105,6 +105,8 @@ class LiveExecEngineConfig(ExecEngineConfig, frozen=True):
         weights imposed by the necessary order status requests.
     qsize : PositiveInt, default 100_000
         The queue size for the engines internal queue buffers.
+    portfolio_bar_updates : bool, default True
```

Comment:
This isn't a live-specific option, as we may want to configure this for backtests as well.
Since we don't have a dedicated portfolio config, consider moving this to `ExecEngineConfig` for now.